chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsReceivedIntegrationTest.kt

```diff
@@ -70,7 +73,8 @@ internal object JoinedRoomsReceivedIntegrationTest : Spek({
             UserSubscriptionDispatcher(
                 joinedRoomApiTypeMapper = joinedRoomApiTypeMapper,
                 joinedRoomsStateDiffer = differ,
-                dispatcher = dispatcher
+                dispatcher = dispatcher,
+                userApiTypeMapper = userApiTypeMapper
```

Comment:
Minor style: Let's group the mappers together in the list

Reply:
👍 Will also update the UserSubscriptionDispatcher to group them together too in the constructor.